x-pack/plugins/synthetics/server/synthetics_service/get_api_key.ts

```diff
@@ -84,8 +86,19 @@ export const generateAndSaveServiceAPIKey = async ({
 
   const apiKeyResult = await security.authc.apiKeys?.create(request, {
     name: 'synthetics-api-key',
-    role_descriptors: {
-      synthetics_writer: serviceApiKeyPrivileges,
+    kibana_role_descriptors: {
```

Comment:
I feel like we should consider separating out the api keys so that we limit priviledges to the scope of what is needed.

The service does not need uptime all permissions.

Users of project monitors do not need ES permissions.

Reply:
This is still granting too many priviledges. We are creating 1 api key with two purposes 1. The service writing to ES and 2. End users being able to hit Uptime apis.